enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/NotificationAcceptanceTest.scala

```diff
@@ -1,629 +0,0 @@
-                                                                                         indexSeekOrScan(
-                                                                                           Set("Person").asJava))))
-  }
-
-  test("warn for unfulfillable index seek when using dynamic property lookup with a single label and regex") {
-    graph.createIndex("Person", "name")
-
-    val result = innerExecute("EXPLAIN MATCH (n:Person) WHERE n['key-' + n.name] =~ 'Foo*' RETURN n")
-
-    result.notifications should equal(Set(INDEX_LOOKUP_FOR_DYNAMIC_PROPERTY.notification(graphdb.InputPosition.empty,
-                                                                                         indexSeekOrScan(
-                                                                                           Set("Person").asJava))))
-  }
-
-  test("warn for unfulfillable index seek when using dynamic property lookup with a single label and IN") {
-    graph.createIndex("Person", "name")
-
-    val result = innerExecute("EXPLAIN MATCH (n:Person) WHERE n['key-' + n.name] IN ['Foo', 'Bar'] RETURN n")
-
-    result.notifications should equal(Set(INDEX_LOOKUP_FOR_DYNAMIC_PROPERTY.notification(graphdb.InputPosition.empty,
-                                                                                         indexSeekOrScan(
-                                                                                           Set("Person").asJava))))
-  }
-
-  test("warn for unfulfillable index seek when using dynamic property lookup with multiple labels") {
-    graph.createIndex("Person", "name")
-
-    val result = innerExecute("EXPLAIN MATCH (n:Person:Foo) WHERE n['key-' + n.name] = 'value' RETURN n")
-
-    result.notifications should contain(INDEX_LOOKUP_FOR_DYNAMIC_PROPERTY.notification(graphdb.InputPosition.empty,
-                                                                                       indexSeekOrScan(
-                                                                                         Set("Person").asJava)))
-  }
-
-  test("warn for unfulfillable index seek when using dynamic property lookup with multiple indexed labels") {
-    graph.createIndex("Person", "name")
-    graph.createIndex("Jedi", "weapon")
-
-    val result = innerExecute("EXPLAIN MATCH (n:Person:Jedi) WHERE n['key-' + n.name] = 'value' RETURN n")
-
-    result.notifications should equal(Set(INDEX_LOOKUP_FOR_DYNAMIC_PROPERTY.notification(graphdb.InputPosition.empty,
-                                                                                         indexSeekOrScan(
-                                                                                           Set("Person", "Jedi").asJava))))
-  }
-
-  test("should not warn when using dynamic property lookup with no labels") {
-    graph.createIndex("Person", "name")
-
-    val result = innerExecute("EXPLAIN MATCH (n) WHERE n['key-' + n.name] = 'value' RETURN n")
-
-    result.notifications shouldBe empty
-  }
-
```

Comment:
The next two test queries also do not appear in Java. The tests exist, but have a wrong query. Probably a copy-paste error?

Reply:
Must be, good catch! Have fixed it now.